src/main/resources/application.properties

```
@@ -0,0 +1,16 @@
+precursor.tolerance=0.5
+fragment.tolerance=0.5
+
+
+# Clustering execution parameters
+threshold.start=0.999F
+threshold.end=0.999F
+cluster.rounds=10
```

Generally 5 is enough
```suggestion
cluster.rounds=5
```